templates/WrongFile.html

```diff
@@ -1,96 +1,11 @@
-<!doctype html>
-<html lang="en">    
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <!-- Link to stylesheet for upload form-->
-        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
-        <!-- Link to stylesheet main.css, not important for this webpage-->
-        <!--link rel="stylesheet" href="Assets/main.css"--> 
-        <!-- Link to stylesheet for the font and the horizontal bar on the top-->
-        <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
-        
+{% extends 'base.html' %}
```

Comment:
It appears this file can be replaced with a [flashed message](https://www.tutorialspoint.com/flask/flask_message_flashing.htm)